Makefile

```diff
@@ -302,3 +302,28 @@ ci.e2e.ansible:
 	make -C test/e2e/ansible e2e.setup
 	WHAT="make -C test/e2e/ansible e2e.run" DOCKER_DEVKIT_DEFAULT_ARGS="--rm --net=host" make devkit.run
 	make -C test/e2e/ansible e2e.clean
+
+
+VERSION = $(shell git describe --tags)
```

Comment:
Can we use the `REPO_REV` that's commented out above? or just the shell from it, since that will allow creating snapshots as well.